include/torch-mlir/Dialect/Torch/Transforms/Passes.td

```diff
@@ -329,4 +329,16 @@ def LowerToBackendContract
   let dependentDialects = ["func::FuncDialect"];
 }
 
+def SatisfiesBackendContract
```

Comment:
Nit: can you name this VerifySatisfiesBackendContract / verify-satisfies-backend-contract -- we like passes to be named with verbs the the beginning.

Reply:
done